license.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Shouldn't this be Github Inc.

https://github.com/electron/electron/blob/master/LICENSE#L1